malariagen_data/anopheles.py

```diff
@@ -6444,34 +6446,33 @@ def plot_haplotype_clustering(
         linkage_method: hapclust_params.linkage_method = hapclust_params.linkage_method_default,
         count_sort: hapclust_params.count_sort = True,
         distance_sort: hapclust_params.distance_sort = False,
-        width: plotly_params.width = 1000,
+        width: plotly_params.width = None,
         height: plotly_params.height = 500,
         show: plotly_params.show = True,
         renderer: plotly_params.renderer = None,
-        render_mode: plotly_params.render_mode = plotly_params.render_mode_default,
-        **kwargs,
+        render_mode: plotly_params.render_mode = "svg",
```

Comment:
I set the default render mode as svg here rather than auto because I tried webgl and it doesn't seem to behave any better in the browser, in fact it seems worse, and also the hover labels for the leaves don't work.